src/components/common/AddLink.tsx

```diff
@@ -0,0 +1,42 @@
+import { makeStyles } from '@mui/styles';
+import { Box, Theme } from '@mui/material';
+import { Icon } from '@terraware/web-components';
+import Link from 'src/components/common/Link';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  addIcon: {
+    fill: theme.palette.TwClrIcnBrand,
+  },
+  disabledIcon: {
+    fill: theme.palette.TwClrBaseGray300,
+  },
+  largeIcon: {
+    height: '20px',
+    width: '20px',
+  },
+}));
+
+type AddLinkProps = {
+  text: string;
+  id?: string;
+  large?: boolean;
+  onClick: (args: any) => void;
+  disabled?: boolean;
+};
+
+export default function AddLink(props: AddLinkProps): JSX.Element {
```

Comment:
nice component!

Reply:
Thanks @charlie-tf - I also added a few others like Link.tsx, TitleDescription.tsx, Card.tsx .. which seem reusable with specs. I can move the useful ones from that list to web-components at some point.